scripts/update_theme_docs.py

```diff
@@ -66,7 +67,30 @@ def markdown_table(rows: list[list[str]]) -> str:
     return "\n".join(lines)
 
 
-def contrast_markdown_table(color_cls: Type, background_color: str) -> list[list[str]]:
+def hexstr_to_rgb(hex_string: str) -> Tuple[int, int, int]:
+    hex_string = hex_string.lstrip("#")
+    r = int(hex_string[0:2], 16)
+    g = int(hex_string[2:4], 16)
+    b = int(hex_string[4:6], 16)
+    return (r, g, b)
+
+
+def make_square_png(hex_color: str):
+    assert hex_color.startswith("#")
+    from PIL import Image
+
+    width, height = 20, 20
+
+    rgb_color = hexstr_to_rgb(hex_color)
+
+    image = Image.new("RGB", (width, height), rgb_color)
+    filename = f"assets/{hex_color[1:]}.png"
```

Comment:
These PNGs should go inside `a11y_pygments`, don't you think? Because that's where the README files that reference these assets are located. Keeps the folder more self-contained.